src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/domain/BlogPost.java

```diff
@@ -95,4 +95,73 @@ public String getApprovalState() {
     public boolean isGoingInNewsletter(final LocalDateTime lastPublicationDate) {
         return publishedDate.isAfter(lastPublicationDate);
     }
+
+    public static BlogPostBuilder builder() {
```

Comment:
@Builder populates uid with null value when not set explicitly so we have to do it manually.